native/skipIntroPlugin.js

```diff
@@ -36,7 +36,7 @@ class skipIntroPlugin {
                 background-color:rgba(25, 25, 25, 0.66);
                 border: 1px solid;
                 border-radius: 0px;
-                display: inline-block;
+                display: inline-block !important;
```

Comment:
The issue was caused by the display: none property being applied to the skip button on every stream, except for the very first one after the app launches (which is why the issue arises on subsequent streams). This PR addresses the problem by using the `!important` rule to override the `display: none` property and ensure the button remains visible.